token/core/identity/msp/idemix/wallet.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package idemix
+
+import (
+	"fmt"
+
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/driver"
+	"go.uber.org/zap/zapcore"
+)
+
+type localMembership interface {
+	DefaultNetworkIdentity() view.Identity
+	IsMe(id view.Identity) bool
+	GetIdentityInfo(label string, auditInfo []byte) (driver.IdentityInfo, error)
+	GetIdentifier(id view.Identity) (string, error)
+	GetDefaultIdentifier() string
+	RegisterIdentity(id string, path string) error
+}
+
+// wallet maps identifiers of different sorts to identities
```

Comment:
wallet maps an identifier to an identity.

Reply:
okay